File: java/src/main/java/com/dddeurope/niuz/newsroom/SubmitArticle.java
package com.dddeurope.niuz.newsroom;

public class SubmitArticle {
    private final String articleId;
    private final String authorId;
    private final String headline;

    public SubmitArticle(String articleId, String authorId, String headline) {
        this.articleId = articleId;
        this.authorId = authorId;
        this.headline = headline;
    }

    public String getArticleId() {
        return articleId;
    }

    public String getAuthorId() {
        return authorId;
    }

    public String getHeadline() {
        return headline;
    }
}

File: dotnet/niuz.application/newsroom/SubmitArticle.cs
namespace niuz.application.newsroom
{
    public class SubmitArticle
    {
        public string ArticleId { get; }
        public string AuthorId { get; }
        public string Headline { get; }

        public SubmitArticle(string articleId, string authorId, string headline)
        {
            ArticleId = articleId;
            AuthorId = authorId;
            Headline = headline;
        }
    }
}